packages/editor-ui/src/components/executions/workflow/WorkflowExecutionsSidebar.vue

```diff
@@ -117,6 +119,12 @@ export default defineComponent({
 	data() {
 		return {
 			filter: {} as ExecutionFilterType,
+			mountedItems: [] as string[],
+			autoScrollDeps: {
+				activeExecutionSet: false,
+				cardsMounted: false,
+				scroll: true,
```

Comment:
Nit: I find this name to be a bit unclear. Can we elaborate a bit more what this flag is doing (better name, or just a comment)?